store/angular.json

```diff
@@ -30,7 +30,7 @@
               "src/assets"
             ],
             "styles": [
-              "src/styles.scss"
+              "src/scss/styles.scss"
```

Comment:
Do not replace global `styles.scss` file

Reply:
Done